modules/date.js

```diff
@@ -1,5 +1,5 @@
 const getDate = () => {
```

Comment:
Hi Abdul, great work implementing this date feature. But there is something you need to do to get the linter error "luxon not defined" out of your way. I suggest you add `/* eslint-disable no-undef */
` at the top of this date.js file. This will disable the linter rule no-undef.